src/components/Programmers.js

```diff
@@ -51,23 +75,27 @@ export default function Programmers() {
           /* Nasty bug! We should map over a slice of state, instead of 'listOfAwesome'.
           We might think: "it works, though!" But if the list of programmers is not state,
           we could never add or edit programmers in the future. The list would be a static thing." */
-          listOfAwesome.map(dev =>
+          list.map(dev =>
             <div className='programmer' key={dev.id}>
-              {dev.name} <button onClick={() => { /* in here set the featured id to be dev.id */ }}>Feature</button>
+              {dev.name} <button onClick={() => { /* in here set the featured id to be dev.id */
+                return setId(dev.id)
```

Comment:
You don't need the return statement here for it to work but it doesn't hurt it being here.